XRTK-Core/Packages/com.xrtk.core/Editor/Templates~/SystemProfile.txt

```diff
@@ -2,7 +2,9 @@
 
 namespace #NAMESPACE#
 {
+    [CreateAssetMenu(menuName = "Mixed Reality Toolkit/#NAME#/#NAME#Profile", fileName = "#NAME#Profile", order = 99)]
```

Comment:
I intentionally left this out before because I didn't think people would want their profiles showing up under the toolkit menu

Reply:
Except without it, the Profile is pointless as you can't create it.
We can always have a guide that tells you to rename it.

Reply:
I'll change it so that the path is not XRTK and is specific to the service being generated.   The Developer can then chose to update/change it.

Reply:
Switched to 

> [CreateAssetMenu(menuName = "#NAMESPACE#/#NAME#/#NAME#Profile", fileName = "#NAME#Profile", order = 99)]